style: align headers icons

ofirc77 opened the pull request:
Hi, this fix is for the following bug - 
https://github.com/hasadna/open-bus-map-search/issues/974

It seems that in the header.css file the '.main-header' template with the field "justify-content" modified the behavior of the headers icons not to be aligned as expected. I change it to "flex-end" that verify that in each language we are using the headers icons bar will be at the end of line which means the other side as expected. Below there is an example of before and after the change.

Before change:
![main_head_bar_in_chrome](https://github.com/user-attachments/assets/81c55078-9656-4af6-899c-791516fb379a)
![main_head_bar_in_firefox](https://github.com/user-attachments/assets/72418cbd-559a-4fd9-b419-3012bc5611fa)

After change:
![fix_head_bar_in_chrome](https://github.com/user-attachments/assets/45f1725f-e16b-48b3-9197-cbbcfe2e3764)
![fix_head_bar_in_firefox](https://github.com/user-attachments/assets/cd1e5234-06ba-4dbe-8234-45bd6774a6dd)

NoamGaash commented:
Thanks!
Have you verified that it looks right on mobile devices?
(visual tests are not executed in the CI pipeline from forked repositories)

ofirc77 commented:
Yes. Checked it on mobile as well.
Hebrew view:
![mobile_view_hebrew](https://github.com/user-attachments/assets/f62b5f32-e195-4a82-b0b9-e3ee2f88f9f4)

English view:
![mobile_view_english](https://github.com/user-attachments/assets/347a8c1f-5dd4-49eb-a9f9-453aff8cffdb)

NoamGaash commented:
In your PR preview, it seems different -
![image](https://github.com/user-attachments/assets/96b2a6b8-2f6c-4934-94d1-91c727e9f8bf)
![image](https://github.com/user-attachments/assets/5f25d7d2-b8e8-494e-a9e4-3d41ee29ca82)

ofirc77 commented:
Hi, updated the MR. Added specific tags for the mobile view. When I checked it as the screenshot you shared I saw the issue.
So I defined the css the order them differently when they are in mobile (also applied for dark mode )

![pic1](https://github.com/user-attachments/assets/162921c2-627d-4476-8482-23c0354792ec)

![pic2](https://github.com/user-attachments/assets/03559cf8-9780-4dec-8862-3331c4918e38)

NoamGaash commented:
Thanks @ofirc77 !
1. could you please verify that you have pushed your code, and that it's passing the linter (`npm run lint`)?
2. did you try to play with the horizontal size of the browser and see that it looks fine on every screen size?

ofirc77 commented:
Yes it passed the lint check and I increased the max-width to 750 to make sure also in horizontal it will look good I played with it a little bit it seems that in some cases it will not function as expected.

NoamGaash commented:
you can see it by visiting the commit statuses section of this pull request:
![image](https://github.com/user-attachments/assets/6b8d0872-2a00-4cba-abf2-2311f591172f)

ofirc77 commented:
@NoamGaash updated the PR according to the project linter and it seems that all tests are passing now

NoamGaash commented on src/layout/header/Header.css:
```diff
@@ -9,6 +9,16 @@
   background: #303230;
 }
 
+@media (width <= 750px) {
```
great, thanks. let's just be consistent with the mobile width
https://github.com/hasadna/open-bus-map-search/blob/f924722ad6e55d0b8b846c0a40eaa449d32c81d4/src/layout/sidebar/sidebar.scss#L17

ofirc77 replied:
Done

NoamGaash reviewed (approved):
Thank you very much! :bus: :100: :1st_place_medal:
